src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -21,7 +21,7 @@ info:
     url: '${license.url}'
 servers:
   #  /api must match servlet-mapping in web.xml
-  - url: '/${project.artifactId}/v1'
+  - url: '${config:openapi.serverurl}'
```

Comment:
It would be super neat if we had a (working) sample image ID in the configuration that was injected to the IIIF & IIP endpoints, so that users of the API could always do `Try -> Execute` without any changes. There is one for IIIF, but it is hardwired in the OpenAPI spec.

Reply:
I'll add this, however the already present id gives a `Image ID not found` when called